silkworm/rpc/core/account_dumper.cpp

```diff
@@ -38,107 +37,119 @@ namespace silkworm::rpc::core {
 
 using db::state::StateReader;
 
+using namespace evmc::literals;
+static constexpr evmc::bytes32 kEmptyHash = 0x0000000000000000000000000000000000000000000000000000000000000000_bytes32;
```

Comment:
Nitpicks:
- `kEmptyHash` -> `kZeroHash`
- move `kZeroHash` in `empty_hashes.hpp`